.github/workflows/qe-ocp.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         with:
           repository: ${{ env.TEST_REPO }}
           path: certsuite
-          ref: main
+          ref: remove_limits
```

Comment:
Temporarily testing based on https://github.com/redhat-best-practices-for-k8s/certsuite/pull/3127